### DISCUSSION TITLE
Task Templates

### DISCUSSION THREAD

**Discussion by tholum** [cline/cline#382] [Feature Requests]
In my codebase, I have a few tasks that I do regularly, and I have some prompts that work well with them, just replacing a few variables, it would be great to be able to have a few variables ( Like Strings, and files )

For example, Two tasks task I am doing 2 - 3 times per day ( Replacing the class name, and the class )

"Fix NodeResolver in @src/recipies/node/node.resolver.ts to use @src/graphql/baseresolver.ts'  like @src/ai/chat/chat.resolver.ts' (see below for file content)  make args in a separate file"

and

"I want RecipeNode @src/recipe/entities/node.entity.ts  to be a Graphql item like I did for @src/ai/entities/chat.entity.ts  and @src/ai/entities/chatmessages.entity.ts"

**Comment by TomLucidor** [comment]
How would task templates be different from a prompt library?

**Comment by gmacon** [comment]
MCP supports prompt templates, but Cline doesn't seem to support them even when registered MCP servers have prompts available. Maybe supporting MCP prompts is a way to address this feature request?

**Comment by d0lwl0b** [comment]
We can jointly implement a "mandatory" standard for LLMs.

see: #5912
